ingestion/src/metadata/profiler/metrics/hybrid/histogram.py

```diff
@@ -125,7 +125,7 @@ def _get_bins(
                 logger.warn(
                     f"An warning occurred while computing the number of bins: {warning}"
                 )
-                num_bins = max_bin_count
+                return 0, bin_width
```

Comment:
what i meant here is that if min or max are inf then let's skip the whole thing entirely. Let's not add that try/except to handle this

Reply:
Got it, Returning 0 returns None at the end which skips the histogram metric indirectly.

Reply:
I will modify the logic to skip the histogram entirely if min / max are `inf / -inf`